src/frontend/src/lib/canisters/signer.canister.ts

```diff
@@ -195,4 +197,27 @@ export class SignerCanister extends Canister<SignerService> {
 
 		throw mapSignerCanisterSendBtcError(response.Err);
 	};
+
+	getSchnorrPublicKey = async (derivationPath: string[]): Promise<Uint8Array | number[]> => {
```

Comment:
This will also need to pass the algorithm.  Maybe move this function into the solana directory?  It  looks as if this is trying to set default values but those default values are specific to solana.  If you move this into the solana directory you don't need to pass the derivation path as an argument as you'll be using the same derivation path every time.  Until you have subaccounts that is!

Reply:
Good one! Done!